code/script.js

```diff
@@ -1,47 +1,205 @@
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/flowerbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
-          <p>${message}</p>
+          <p class="message">${message}</p>
         </div>
       </section>
-    `
+     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+//Greeting function
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there, What's your name?`, 'bot');
+  
+}
+// Function that takes adds event listener to the form submission
+const initFncn = () => {
+  const inputFromForm = document.getElementById('input-ID');
+  theWholeForm.addEventListener('submit', (event) => {
+    event.preventDefault();
+    questionNumber = 1;
+    const name = inputFromForm.value;
+    questionHandler (name);
+  })
+}
+//Functions that plays sound effects
+const playSoundUser = () => {
+  const soundEffectUser = document.getElementById('message-sound')
+  soundEffectUser.play()
+}
+const playSoundBot = () => {
+  const soundEffectBot = document.getElementById('bot-sound')
+  soundEffectBot.play()
+}
+// Function that shows flower arrangement choices
+const showFlowerArrangement = (name) =>{
+  questionNumber++; 
+  showMessage (`Hello ${name}, What flower arrangement would you like today?`,'bot'); 
+  theWholeForm.innerHTML = `
+    <button class="arrangement-button" id="bouquet" type="button">Bouquet</button>
+    <button class="arrangement-button" id="basket" type="button">Basket</button>
+    <button class="arrangement-button" id="box" type="button">Box</button>
+  `
+  playSoundBot();
+  document.getElementById("bouquet").addEventListener('click', () => {
+    questionHandler('bouquet');
+  });
+  document.getElementById("basket").addEventListener('click', () => {
+    questionHandler('basket');
+  });
+  document.getElementById("box").addEventListener('click', () => {
+    questionHandler('box');
+  }); 
+};
+
+//Function that shows different types of flower arrangements depending on the choice of bouquet/basket/box
+const showTypeOfArrangement = (arrangement) => {
+  questionNumber++;
+  showMessage (`Wonderful! I always loved flowers in a ${arrangement}. We have the following options for the ${arrangement}`, 'bot');
+  playSoundBot();
+  if (arrangement === 'bouquet'){
+    theWholeForm.innerHTML = `
+      <select id="selector" class="general-selector">
+        <option value="" selected disabled>&#8681 Choose bouquet</option>  
+        <option value="Cascade bouquet">Cascade bouquet</option>
+        <option value="Round bouquet">Round bouquet</option>
+        <option value="Hand-Tied bouquet">Hand-Tied bouquet</option>
+      </select>  `
```

Comment:
You are repeating this code a few times. Maybe it would be a good idea to make a template and call it when needed? 😸